app/controllers/application_controller.rb

```diff
@@ -10,4 +10,12 @@ def enforce_basic_auth
       BasicAuthenticable.authenticate(username, password)
     end
   end
+
+  def current_user
+    @current_user ||= User.find(session[:auth_user][:user_id])
```

Comment:
Will `current_user` play a role in creating records in the future? No tables currently have a `user_id` foreign key.

Reply:
It probably will. We'll be adding some sort of audit trail at some point so it will definitely be needed there. It's mostly for authorization at the moment.